.rubocop.yml

```
@@ -0,0 +1,14 @@
+inherit_mode:
+  merge:
+    - Exclude
+
+require:
+  - rubocop-performance
+  - rubocop-rspec
+  - standard
+
+inherit_gem:
+  standard: config/base.yml
```

```suggestion
inherit_gem:
  standard: config/base.yml
  standard-performance: config/base.yml
  standard-custom: config/base.yml
```

This pulls in the standard performance extensions (and its custom extensions).